qiskit_machine_learning/algorithms/trainable_model.py

```diff
@@ -32,17 +32,20 @@
 from .serializable_model import SerializableModelMixin
 
 
+POINT = Union[float, np.ndarray]
```

Comment:
Is this used somewhere - maybe I missed it. I recall optimizer in Terra defines this - but its needed here too?

Reply:
It is not used, now removed.